src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
```

Comment:
I can't use `run_command` in this situation, because I'd like for it to display the Git branch here, so that this step can be listed and verified in the subsequent `Then it runs the Git commands` step.

This has several benefits:
* it is important on which branch the `cd` operation happens, since the folder cd'ed out of only exists on a particular branch
* when the `cd` command happens in the order of steps is important, because if it happens at the wrong time it leaves the file system in limbo
* it allows to document when exactly we `cd` back into this folder at the end

This means the `Then it runs the Git commands` step probably has to be renamed to `Then it runs the following commands in my workspace` to account for the fact that we now also run non-Git commands in the current workspace. 

Since this is a larger operation, I'd like to do it separately. 

I'm open to doing this differently, but I don't see a way to verify the `cd` command in the spec in another similarly clean way right now. If you see one, please outline it.